.github/workflows/main.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
     - name: run tests (macos)
       if: matrix.os == 'macos-latest'
-      run: go test -v ./internal/... && go test -c && sudo ./rsync.test -test.v
+      run: go test -v ./pkg/... && go test -c && sudo ./rsync.test -test.v
```

Comment:
GitHub actions workflow now pass in my repo

Whether this is an appropriate fix depends on whether the blanket internal -> pkg rename is ok for now